src/Build/Resources/Strings.resx

```diff
@@ -1271,6 +1271,10 @@
     <value>MSB4067: The element &lt;{0}&gt; beneath element &lt;{1}&gt; is unrecognized.</value>
     <comment>{StrBegin="MSB4067: "}</comment>
   </data>
+  <data name="PropertyOutsidePropertyGroupInTarget" xml:space="preserve" Condition="$([MSBuild]::AreFeaturesEnabled('17.6'))">
+    <value>MSB4070: The property &lt;{0}&gt; beneath target &lt;{1}&gt; is unrecognized. Please define property beneath property group</value>
```

Comment:
It looks like `MSB4070` is already taken (`The schema "{0}" is not valid.`). @baronfel, how would you feel about continuing to use MSB4067, but making the message more specific, like

```suggestion
    <value>MSB4067: The element &lt;{0}&gt; beneath target &lt;{1}&gt; is unrecognized. Properties must be inside a &lt;PropertyGroup&gt; element.</value>
```

Reply:
That change makes sense to me - clarification and direction in an error message is a fine kind of change to make.